addons/mail/static/tests/chat_window/chat_window_manager_tests.js

```diff
@@ -175,12 +179,20 @@ QUnit.test(
         await click(".o_menu_systray i[aria-label='Messages']");
         await click(".o-mail-NotificationItem-name", { text: "channel-D" });
         await contains(".o-mail-ChatWindowHiddenToggler", { text: "2" });
-        await contains(".o-mail-ChatWindow-name:eq(0)", { text: "channel-A" });
-        await contains(".o-mail-ChatWindow-name:eq(1)", { text: "channel-D" });
-        await click(
-            ".o-mail-ChatWindow-header:contains(channel-D) .o-mail-ChatWindow-command[title='Close Chat Window']"
-        );
-        await contains(".o-mail-ChatWindow-name:eq(0)", { text: "channel-A" });
-        await contains(".o-mail-ChatWindow-name:eq(1)", { text: "channel-C" });
+        await contains(":nth-child(1 of .o-mail-ChatWindow) .o-mail-ChatWindow-name", {
+            text: "channel-A",
+        });
+        await contains(":nth-child(2 of .o-mail-ChatWindow) .o-mail-ChatWindow-name", {
+            text: "channel-D",
+        });
+        await click(".o-mail-ChatWindow-command[title='Close Chat Window']", {
+            parent: [".o-mail-ChatWindow-header", { text: "channel-D" }],
+        });
+        await contains(":nth-child(1 of .o-mail-ChatWindow) .o-mail-ChatWindow-name", {
+            text: "channel-A",
+        });
+        await contains(":nth-child(2 of .o-mail-ChatWindow) .o-mail-ChatWindow-name", {
+            text: "channel-C",
+        });
```

Comment:
A bit sad it's not as readable as before.
Hopeful we'll eventually have a relax `text:` like `:contains`? so that we can simply target `.o-mail-ChatWindow` rather than explicitly `-name` part